### ISSUE TITLE
--recursive in compress_fast5

### ISSUE THREAD

**Issue by pterzian** [nanoporetech/ont_fast5_api#28]
Hi,

Could you confirm that the `--recursive` option doesn't take any boolean parameter instead of what is stated in the example usage :
```
compress_fast5 --input_path /data/uncompressed_reads --save_path /data/compressed_reads
    --compression vbz --recursive True --threads 40
```
This returns the `unrecognized arguments: True` error. 

Does calling the option as the following set --recursive to True and not to False (it is stated default=False)   :  
```
compress_fast5 --input_path /data/uncompressed_reads --save_path /data/compressed_reads
    --compression vbz --recursive --threads 40
```

Paul

**Comment by fbrennen** [comment]
Hi @pterzian -- apologies, you're entirely correct: `--recursive` does not take an argument, and by default is disabled. Your second example correctly enables recursive folder navigation.

We'll get this fixed.

**Comment by davidnewman02** [comment]
This has been fixed in v3.0
